components/dashboards/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/WidgetMetadataProvider.java

```diff
@@ -58,6 +58,9 @@
      */
     Set<WidgetMetaInfo> getAllWidgetConfigurations() throws DashboardException;
 
+
+    void delete(String widgetId) throws DashboardException;
```

Comment:
Please add javadoc for this method

Reply:
Fixed by: c7f9a9abd89b29901f03eaf82fab6dbf58f3ad8f